package.json

```diff
@@ -1,26 +1,56 @@
 {
-  "name": "webpack-demo",
+  "name": "clear",
   "version": "1.0.0",
-  "description": "",
+  "description": "<a name=\"readme-top\"></a>",
   "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --env=jsdom",
     "start": "webpack serve --open",
-    "build": "webpack",
-    "deploy": "gh-pages -d dist"
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Muhammad0602/Leaderboard.git"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Muhammad0602/Leaderboard.git/issues"
+  },
+  "homepage": "https://github.com/Muhammad0602/Leaderboard.git#readme",
   "devDependencies": {
+    "@babel/plugin-transform-modules-commonjs": "^7.21.2",
+    "babel-eslint": "^10.1.0",
     "css-loader": "^6.7.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "gh-pages": "^5.0.0",
+    "hint": "^7.1.4",
     "html-webpack-plugin": "^5.5.0",
+    "jest-environment-jsdom": "^29.5.0",
```

Comment:
- [ ] [OPTIONAL]
Good job with the changes, however, I still had to add the lines below in the _package.json_ after installation in order for the jest to run correctly:
```javascript
"jest": {
    "testEnvironment": "jsdom"
  },
```